math.asm

```diff
@@ -163,3 +163,25 @@ end:    INX
     sta MSB+1,x
     inx
     rts
+
+    +BACKLINK "abs", 3
+ABS:
+    lda MSB,x
+    bmi +
+    rts
++   jmp NEGATE
```

Comment:
cool! to make it a yota faster, I guess one could move NEGATE close enough to branch to it directly?

Reply:
...that would also save a few bytes, to replace the `jmp NEGATE ` calls with `b?? NEGATE`

Reply:
Done.  Indeed saves a few bytes.